### ISSUE TITLE
UI for json-rules-engine - "Json-rule-editor"

### ISSUE THREAD

**Issue by vinzdeveloper** [CacheControl/json-rules-engine#211]
First of all - its not related to Issue :)

I am using json-rules-engine and configured my business rules under json files in my web application . Since, business logics are getting higher, i am facing issues in terms of readability despite maintaining under separate files based on business use cases.

so, developed simple intuitive user interface also known as json rule editor to manage the business rules and conditions. it actually gives better representation of our business rules besides we can validate the rules here. 

It may sounds extra steps getting added into the work. However it will be certainly helpful to people whom would want to manage business rules via user interface like me :)

Github Location:

https://github.com/vinzdeveloper/json-rule-editor

Documentation is also available in github pages

https://vinzdeveloper.github.io/json-rule-editor/docs

App is deployed here to let people use directly instead of installing locally

https://www.json-rule-editor.com/

Possibly, I might miss some functionalities of json-rules-engine in this json-rule-editor. If you guys find useful and come up with any suggestions or improvement, please do let me know - Happy to improvise it!

<img width="1676" alt="rule editor 1" src="https://user-images.githubusercontent.com/61467683/82750272-dc0a0e80-9da6-11ea-98ec-3549bf530f4d.png">
<img width="1680" alt="rule editor 2" src="https://user-images.githubusercontent.com/61467683/82750274-dd3b3b80-9da6-11ea-96eb-434a6a1a9bc1.png">

**Comment by jorchg** [comment]
@vinzdeveloper That's a cool piece of work. I think it may be suitable for my needs. In fact, me and my team were discussing the creation of something similar.
I will be reviewing it and if it does accomplish our needs, maybe contribute with you in the development.

Thank you! 👍🏻

**Comment by CacheControl** [comment]
thanks @vinzdeveloper! This is neat! I played around with the demo and I really liked all of it, especially the visualization part! The question of a UI for json-rules-engine comes up all the time, so I'm sure your project will help a lot of folks.

I wanted to keep this issue open until I had a chance to add a reference in the README - I added a [Related Projects](https://github.com/CacheControl/json-rules-engine#related-projects) section to make this discoverable (feel free to submit a PR if you'd like to tweak anything). Thanks again!
